src/Apps/Artist/Routes/Overview/Components/RecommendedArtist.tsx

```diff
@@ -0,0 +1,188 @@
+import { EntityHeader, Sans, Spacer } from "@artsy/palette"
+import { RecommendedArtist_artist } from "__generated__/RecommendedArtist_artist.graphql"
+import { track } from "Artsy/Analytics"
+import * as Schema from "Artsy/Analytics/Schema"
+import { SystemContext } from "Artsy/SystemContext"
+import { FillwidthItem } from "Components/Artwork/FillwidthItem"
+import { FollowArtistButtonFragmentContainer as FollowArtistButton } from "Components/FollowButton/FollowArtistButton"
+import { Carousel } from "Components/v2"
+import { stringify } from "qs"
+import React, { FC, useContext } from "react"
+import { createFragmentContainer, graphql } from "react-relay"
+import { data as sd } from "sharify"
+import { get } from "Utils/get"
+
+interface RecommendedArtistProps {
+  artist: RecommendedArtist_artist
+}
+const HEIGHT = 150
+
+// TODO: Remaining tasks
+/*
+ * find out tracking constants to use
+ * Move handleOpenAuth/openMobileAuth/openDesktopAuth to shared location
+ * Buy now/make offer labels
+ * Possibly extract <followArtistButton> to a shared location (it was a lot of copypasta that just worked)
+ * Tests
+ */
+
+const handleOpenAuth = (mediator, artist) => {
+  if (sd.IS_MOBILE) {
+    openMobileAuth(artist)
+  } else if (mediator) {
+    openDesktopAuth(mediator, artist)
+  } else {
+    window.location.href = "/login"
+  }
+}
+
+const openMobileAuth = artist => {
+  const params = stringify({
+    action: "follow",
+    contextModule: Schema.ContextModule.RecommendedArtists,
+    intent: "follow artist",
+    kind: "artist",
+    objectId: artist.id,
+    signUpIntent: "follow artist",
+    trigger: "click",
+    entityName: artist.name,
+  })
+  const href = `/sign_up?redirect-to=${window.location}&${params}`
+
+  window.location.href = href
+}
+
+const openDesktopAuth = (mediator, artist) => {
+  mediator.trigger("open:auth", {
+    mode: "signup",
+    copy: `Sign up to follow ${artist.name}`,
+    signupIntent: "follow artist",
+    afterSignUpAction: {
+      kind: "artist",
+      action: "follow",
+      objectId: artist.id,
+    },
+  })
+}
+
+@track({
+  context_module: Schema.ContextModule.RecommendedArtists,
+})
+class RecommendedArtistWithTracking extends React.Component<
+  RecommendedArtistProps
+> {
+  @track({
+    type: Schema.Type.Thumbnail,
+    action_type: Schema.ActionType.Click,
+  })
+  trackArtworkClicked() {
+    // noop
+  }
+
+  render() {
+    return (
+      <RecommendedArtist
+        {...this.props}
+        onArtworkClicked={this.trackArtworkClicked.bind(this)}
+      />
+    )
+  }
+}
+
+const RecommendedArtist: FC<
+  RecommendedArtistProps & { onArtworkClicked: () => void }
```

Comment:
And then this "extension" of RecommendedArtistProps is a result of me splitting the component into a slim class for tracking + a function that does most of the stuff. I can be convinced that I should create a new "type" for the props here, but this was quick, easy, and made sense.

Reply:
@alloy - thoughts on a `useTracking` hook? It would be nice to move away from classes completely.

Reply:
https://artsyproduct.atlassian.net/browse/PLATFORM-1216